squadrons.md

```diff
@@ -23,6 +23,7 @@
   | Charles | USA | UTC-5 | ReachableCEO | [reachableceo](https://twitter.com/reachableceo) | [reachableceo](https://github.com/reachableceo) | N/A | 0 |
   | Nearly | USA | UTC-4 | Nearly | [NearlyNeverHere](https://twitter.com/NearlyNeverHere) | [NearlyAlwaysThere](https://github.com/NearlyAlwaysThere) | N/A | 0 |
   | James | USA | UTC-5 | jamesscaggs | [jamesscaggs](https://twitter.com/jamesscaggs) | [jamesscaggs](https://github.com/jamesscaggs) | [jamesscaggs](https://www.linkedin.com/in/jamesscaggs) | 0 |
+  | Silvestre | EU | CET | silvestrevivo | [silvestrevivo](https://twitter.com/silvestrevivo) | [silvestrevivo](https://github.com/silvestrevivo) | [silvestrevivo](https://www.linkedin.com/in/silvestre-vivo-1a6843bb) | 0 |
```

Comment:
Hi Silvestre! The info looks great.

Would you move down to the EU squadron, please?
We're going to try a consensus process between 2 squadrons at some point.

And also do UTC+1? It's easier to understand for scheduling.

Thank you very much!